app/src/main/java/com/mertceyhan/bitcoinmarket/features/market/di/AppModule.kt

```diff
@@ -0,0 +1,33 @@
+package com.mertceyhan.bitcoinmarket.features.market.di
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.room.Room
+import com.mertceyhan.bitcoinmarket.features.market.data.local.MarketDatabase
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object AppModule {
+
+    @Provides
+    @Singleton
+    fun provideMarketDatabase(
+        @ApplicationContext context: Context
+    ): MarketDatabase =
+        Room.databaseBuilder(context, MarketDatabase::class.java, MarketDatabase.DB_NAME)
+            .build()
+
+    @Singleton
+    @Provides
+    fun provideSharedPreferences(
+        @ApplicationContext context: Context
+    ): SharedPreferences = context.getSharedPreferences(
+        "asd", Context.MODE_PRIVATE
```

Comment:
Could you change the naming of `SharedPreferences`? The name `asd` doesn't make any sense to me.